docs/Installation-Windows.md

```diff
@@ -0,0 +1,190 @@
+```
+
+You may be asked to install new packages. Type `y` and press enter _(make sure you are connected to the internet)_. You must install these required packages. The new Conda environment is called ml-agents and uses Python version 3.6.
+
+<p align="center">
+    <img src="images/conda_new.PNG" 
+        alt="Anaconda Install" 
+        width="500" border="10" />
+</p>
+
+To use this environment, you must activate it. _(To use this environment In the future, you can run the same command)_. In the same Anaconda Prompt, type in the following command:
+
+```
+conda activate ml-agents
+```
+
+You should see `(ml-agents)` prepended on the last line.
+
+Next, install `tensorflow` and `tensorflow-gpu`. Install these packages using `pip` - which is a package management system used to install Python packages. In the same Anaconda Prompt, type in the following command _(make sure you are connected to the internet)_:
+
+```
+pip install tensorflow tensorflow-gpu
+```
+
+## Step 3: Install Required Python Packages
+
+ML-Agents depends on a number of Python packages. Use `pip` to install these Python dependencies.
+
+If you haven't already, clone the ML-Agents Github repository to your local computer. You can do this using Git ([download here](https://git-scm.com/download/win)) and running the following commands in an Anaconda Prompt _(if you open a new prompt, be sure to activate the ml-agents Conda environment by typing `activate ml-agents`)_:
+
+```
+git clone git@github.com:Unity-Technologies/ml-agents.git
+```
+
+If you don't want to use Git, you can always directly download all the files [here](https://github.com/Unity-Technologies/ml-agents/archive/master.zip).
+
+In our example, the files are located in `C:\Downloads`.  After you have either cloned or downloaded the files, from the Anaconda Prompt, change to the python directory inside the ML-agents directory:
+
+```
+cd C:\Downloads\ml-agents\python
+```
+
+Make sure you are connected to the internet and then type in the Anaconda Prompt:
+
+```
+pip install .
+```
+
+This will complete the installation of all the required Python packages to run ML-Agents.  
+
+## (Optional) GPU Training using ML-Agents 
+
+Not required to use v0.3 for ML-Agents.  This is a guide for advanced users who want to train using GPUs.  Additionally, you will need to check if your GPU is CUDA compatible.  Please check Nvidia's page [here](https://developer.nvidia.com/cuda-gpus).
+
+## (Optional) Step 1: Install Nvidia CUDA toolkit
+
+<a href="https://developer.nvidia.com/cuda-toolkit-archive" target="_blank">Download</a> and install the CUDA toolkit from Nvidia's archive.  The toolkit includes GPU-accelerated libraries, debugging and optimization tools, a C/C++ compiler and a runtime library and is needed to run ML-Agents.  You can select the latest or previous releases.  In this guide, we are using version 9.1.85.3 ([direct link](https://developer.nvidia.com/compute/cuda/9.1/Prod/patches/3/cuda_9.1.85.3_windows)).  
+
```

Comment:
ML-Agents does not work with CUDA 9 because it requires tensorflow 1.5 which is not supported in the internal brain.

Reply:
done